pkgs/tools/virtualization/cloudmonkey/default.nix

```diff
@@ -0,0 +1,25 @@
+{ python2Packages, lib }:
```

Comment:
if there's no tests, add `doCheck = false;` and include a comment explaining why they're disabled